Meadowlark-js/packages/meadowlark-core/src/handler/OAuthHandler.ts

```diff
@@ -6,7 +6,7 @@
 import querystring from 'querystring';
 import secureRandom from 'secure-random';
 import { Logger } from '../Logger';
-import { client1, client2, client3, client4 } from '../security/HardcodedCredential';
+import { admin1, client1, client2, client3, client4 } from '../security/HardcodedCredential';
```

Comment:
I thought client4 was already an admin? (If so, perhaps it should be renamed)

Reply:
Ooooh, right, bootstrap admin compared to `client4` as server host account. NVM.

Reply:
Ah, didn't see that. Will just rename.